### ISSUE TITLE
Question about syncnet

### ISSUE THREAD

**Issue by laxyon** [yuangan/EAT_code#43]
Hi, i have a question about the syncnet for img-st2 training, is that finetuned from one like in wav2lip or you trained it from scratch? and what about the dataset for it?

**Comment by yuangan** [comment]
Hi, we finetuned wav2lip. Our model is trained on a combination of MEAD and part of the Vox dataset, with MEAD serving as the test set. You can find the guidance [here](https://github.com/Rudrabha/Wav2Lip?tab=readme-ov-file#training-the-expert-discriminator)
Our training data undergoes a cropping process, retaining only facial regions, as shown below:

![image](https://github.com/user-attachments/assets/7270801e-0707-44d3-b7fd-822fc1166888)

By the way, the training is challenging to converge and requires multiple attempts with different parameter combinations.

**Comment by laxyon** [comment]
Thanks for your replying, i just tried to train my a2kp model and i realized when i used your expert_sync it was easier for convergence, and i got better result than original wav2lip expert_sync, so im a little bit curious about how you trained it. how about your eval loss during your finetuning?
